src/Server.js

```diff
@@ -49,6 +50,11 @@ function middleware(bundler) {
       } else {
         // Otherwise, serve the file from the dist folder
         req.url = req.url.slice(bundler.options.publicURL.length);
+        // If we're serving an html bundle, and the requested path has not file
+        // extension, serve the index.html. Otherwise go to the file server.
+        if (bundler.mainAsset.type === 'html' && path.extname(req.url) == "") {
```

Comment:
This would break any file that doesn't have an extension?

Reply:
Currently only HTML is served, so it wouldn't change the behavior. But it's true that it will not serve files without extensions - but I'd argue that is desired behavior for single page apps.

Reply:
I need to verify my domain for Apple Pay for my frontend Stripe Elements app, and this requires serving a static file without a file extension, so there is a use case for this.